src/test/java/spoon/test/prettyprinter/TestSniperPrinter.java

```diff
@@ -398,16 +418,7 @@ private void testSniper(String testClass, Consumer<CtType<?>> transformation, Bi
 	private static Launcher createLauncherWithSniperPrinter() {
 		Launcher launcher = new Launcher();
 		launcher.getEnvironment().setPrettyPrinterCreator(() -> {
-			SniperJavaPrettyPrinter printer = new SniperJavaPrettyPrinter(launcher.getEnvironment());
-			printer.setPreprocessors(Collections.unmodifiableList(Arrays.<Processor<CtElement>>asList(
-					//remove unused imports first. Do not add new imports at time when conflicts are not resolved
-					new ImportCleaner().setCanAddImports(false),
-					//solve conflicts, the current imports are relevant too
-					new ImportConflictDetector(),
-					//compute final imports
-					new ImportCleaner()
-			)));
-			return printer;
```

Comment:
Removed as it interfered with my test cases (removed unused imports), and no other test cases actually rely on this functionality. Very unclear why it's here.